### PR TITLE
mkcloud: Disable want_ldap checks for known-broken cloud sources

### PR DESCRIPTION
**Pull request by sayalilunkad**

### CODE REVIEW

File: scripts/jenkins/ci.suse.de/openstack-mkcloud.xml [hunk 1]
```diff
@@ -253,11 +253,18 @@ case $cloudsource in
     GM4|GM4+up|GM5|GM5+up)
         echo "Not setting want_ldap=1"
         ;;
+    develcloud6)
+        echo "Setting want_ldap=0 for upgrade jobs until hybrid backend is ported to Mitaka"
+        [[ $mkcloudtarget == "*plain_with_upgrade*" ]] &amp;&amp; export want_ldap=0
```
Comment (bmwiedemann) [inline scripts/jenkins/ci.suse.de/openstack-mkcloud.xml]: or [[ $mkcloudtarget =~ upgrade ]] to also cover the case when someone gave the cloudupgrade step explicitly.

**Comment by MaximilianMeister** [comment]
triggered an upgrade gating, after addressing @bmwiedemann it looks good +1

**Comment by rhafer** [comment]
+1

**Comment by aplanas** [comment]
+1

**Pull request merged**